tests/components/hooks.test.ts

```diff
@@ -396,6 +396,35 @@ describe("hooks", () => {
       ]);
     });
 
+    test("effect can depends on stuff in dom", async () => {
```

Comment:
depends => depend

Also not sure what this tests. It should probably test that the dependency argument is null on first effect run (mounted) and has the element on rerun